MCprep_addon/util.py

```diff
@@ -184,8 +183,17 @@ def obj_copy(base, context=None, vertex_groups=True, modifiers=True):
 	return new_ob
 
 
-def min_bv(version):
+def max_bv(version, *, up_to=None):
```

Comment:
Per comment in other PR, isn't this about the equiv to inverting the output of min_bv? I'd rather not add a new function.